### PR TITLE
Setting up CSD maintenance branch of bretthoerner/dropwizard-scala 0.6.2-1

### PR DESCRIPTION
**Pull request by markhamstra**
We're now maintaining our own fork of dropwizard-scala since Coda Hale is no longer supporting the library.  Brett Hoerner has picked it up, and our fork adds a few CSD tweaks to his.

This PR is on a maintenance branch from Brett's current release: 0.6.2-1 (based on dropwizard 0.6.2).  By bumping jackson.version up to 2.2.2 (which we've been using for awhile throughout our stack), we can build a Scala 2.9.3 version of dropwizard-scala and eliminate our current need to special case a 2.9.2 version.

Other than that, this PR is all about setting up a Maven build.  I think I've included everything that we need in the pom, and nothing that we don't; but please give it a look.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,181 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.clearstorydata</groupId>
+  <artifactId>dropwizard-scala_2.9.3</artifactId>
+  <packaging>jar</packaging>
+  <description>dropwizard-scala</description>
+  <version>0.6.2-1</version>
```
Comment (mbautin) [inline pom.xml:8]: Should the version have `csd` in it?

Comment (markhamstra) [inline pom.xml:8]: Not sure, since I took the approach of changing the groupId to com.clearstorydata.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+          <skipTests>true</skipTests>  <!-- uses scalatest instead -->
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.scalatest</groupId>
+        <artifactId>scalatest-maven-plugin</artifactId>
+        <version>1.0-M2</version>
+        <configuration>
+          <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
+          <junitxml>.</junitxml>
```
Comment (mbautin) [inline pom.xml]: What is the current directory in this context? Could this be changed to `${project.build.directory}`?

**Comment by mbautin** [comment]
Looks good, a couple of minor comments inline.

File: pom.xml (hunk 1 above)
Comment (mbautin) [inline pom.xml:8]: OK, sounds good.

**Pull request merged**